ACadSharp.Tests/ACadSharp.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+	<TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Can you match the indentation? I'll look for a way to configure it in the .editorconfig

Reply:
Yes.  I accidentally replaced it with a tab...

Reply:
Just confirming, the .editorconfig file has the project using tabs `indent_style = tab`.  I'm showing tabs, but the original file was spaces.  Can you please confirm tabs are what is intended?

Reply:
Correct, that's the right configuration.